COMMUNICATION.md

```diff
@@ -2,22 +2,62 @@
 
 ## Document purpose
 
-OpenTelemetry has a [Code of Conduct](https://github.com/open-telemetry/community/blob/master/code-of-conduct.md) which primarily pertains to "bannable" offenses: things like harassment, personal attacks, and so on. Certainly OpenTelemetry does not tolerate any such behavior.
-
-And while we all deserve to collaborate without fear of overt harassment and personal attacks, that is a very low bar! This document is more aspirational and has fewer "teeth" (i.e., it's not about "banning" people), but it's also very important: it describes **how we wish to communicate with each other** and **how we aspire to create an environment that cultivates trust**, even in the face of inevitable technical and organizational disagreement.
-
-Finally, this is **a living document** and should be adapted over time as the OpenTelemetry project evolves. At any time, there are certainly errors of omissions or even simple clarifications we can make to the wording here, and any and all are welcome to propose such changes.
-
-Now, without further ado:
+OpenTelemetry has a [Code of
```

Comment:
I'm fine with the spirit of this change... I just want to make a few very minor grammatical fixes. :) Also, after reviewing the [CNCF code of conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md) in more detail, it seems as though there's nothing about "banning" or prohibiting certain individuals from participation... it's all about commits/comments/etc. So I adjusted the wording accordingly.

How about:

```
OpenTelemetry has a [Code of
Conduct](https://github.com/open-telemetry/community/blob/master/code-of-conduct.md)
where maintainers and community members pledge to foster a welcoming and open
community. The code of conduct cites examples of unacceptable behavior by
participants – like harassment, trolling or personal attacks – that would lead to
removal, revision, or rejection of contributions to the project.
```

Reply:
works for me